docs/docs-content/byoos/capi-image-builder/capi-image-builder.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_label: "CAPI Image Builder"
+title: "CAPI Image Builder"
+description: "Learn how to use the CAPI Image Builder project to create images for Palette and VerteX."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["operating system", "byoos", "capi iamge builder"]
+---
+
+The CAPI Image Builder is a tool we developed and maintain to reduce the challenges associated with creating images for
+Kubernetes clusters. It is based on the upstream
+[Kubernetes Image Builder (KIB)](https://image-builder.sigs.k8s.io/introduction.html) project.
+
+The tool includes all the dependencies required to build FIPS and non-FIPS images within a Docker container, eliminating
+the need to install any dependencies on your local machine. This is particularly useful when working in air-gapped
```

Comment:
```suggestion
the need to install any dependencies on your local machine. This is particularly useful when working in airgapped
```

We use airgap as a single word. I used to write like this when i didn't know but we've gone and corrected all of them

Reply:
Omg, I looked at our word list and still wrote it wrong 😶‍🌫️